systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -74,11 +79,34 @@
 
 @Tag(REGRESSION)
 @Tag(CRUISE_CONTROL)
+@SuiteDoc(
+    description = @Desc("Verify the behavior and correct functionality of Cruise Control in various scenarios using Kafka."),
```

Comment:
If you want to describe the suite, I guess it should be something longer or something that more describes the suite. Because description of every suite looks completely similar.

Reply:
Updated the description, hope it's better now.